lib/routes/utils/totp.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const errors = require('../../error')
+
+module.exports = (log, config, db) => {
+
+  return {
+    /**
+     * Helper function to check if the specified account has a verified
+     * and enabled TOTP token.
+     *
+     * @param account
+     * @returns boolean
+     */
+    hasTotpToken(account) {
```

Comment:
`hasTotpToken` was being used in a few places so opted to pull into a TOTP utilities module. There are probably a few more things can go into this helper, but in the meantime it is just this.